app/styles/components/_inplace-textarea.scss

```diff
@@ -0,0 +1,14 @@
+.inplace-textarea {
```

Comment:
This should be in _editinplace.scss since it is common styling for all inplace elements.